crates/precompile/build.rs

```diff
@@ -0,0 +1,115 @@
+use std::fs;
+use std::path::Path;
+
+const BYTES_PER_G1_POINT: usize = 48;
+const BYTES_PER_G2_POINT: usize = 96;
+
+fn main() {
+    generate_kzg_settings();
+}
+
+fn generate_kzg_settings() {
+    let in_path = "src/kzg/trusted_setup.txt";
+    let out_path = "src/kzg/generated_settings.rs";
+
+    let in_path = Path::new(in_path);
+    println!("cargo:rerun-if-changed={}", in_path.display());
+    assert!(in_path.exists());
+    let contents = format_kzg_settings(in_path);
+    fs::write(out_path, contents).unwrap();
+}
```

Comment:
makes sense - generating consts here makes creating the default kzgsettings easy, and does not require reading the file on startup etc